sbncode/CAFMaker/FillReco.cxx

```diff
@@ -188,6 +188,25 @@ namespace caf
     srsbndcrttrack.tof      = track.ToF();
   }
 
+  void FillSBNDCRTVeto(const art::Ptr<sbnd::crt::CRTVeto> &veto,
+		       const std::vector<art::Ptr<sbnd::crt::CRTSpacePoint>> &points,
+                       caf::SRSBNDCRTVeto &srsbndcrtveto,
+                       bool allowEmpty)
+  {
+    srsbndcrtveto.V0     = veto->V0();
+    srsbndcrtveto.V1     = veto->V1();
+    srsbndcrtveto.V2     = veto->V2();
+    srsbndcrtveto.V3     = veto->V3();
+    srsbndcrtveto.V4     = veto->V4();
+
+    // add the CRTSpacePoint associations to the SR Veto
+    for(auto const& sp : points) {
+      srsbndcrtveto.sp_position.emplace_back(sp->X(), sp->Y(), sp->Z());   
+      srsbndcrtveto.sp_time.emplace_back(sp->Ts0()); /// ns for SBND CRT SpacePoints   
+      srsbndcrtveto.sp_pe.emplace_back(sp->PE());   
```

Comment:
I _suggest_ the use of good old `push_back()` when the purpose is just to copy the argument into the vector. `emplace_back()` works too, but it has the slightly different purpose of creating a new object into the vector rather than copying an existing one.
```suggestion
      srsbndcrtveto.sp_time.push_back(sp->Ts0()); /// ns for SBND CRT SpacePoints   
      srsbndcrtveto.sp_pe.push_back(sp->PE());   
```